Tests/RouterTests.swift

```diff
@@ -77,7 +88,95 @@ class RouterTests: QuickSpec {
             RouterTestServer.disable()
             Router.disableAll()
         }
-        
+
+        describe("Describe Cancelling requests") {
```

Comment:
Hmmm .... why no describe here?
Shouldn't `describe` be used to group the cancellation specific tests?
How should the three `it(...)` statements be grouped instead?